sklearn/manifold/spectral_embedding_.py

```diff
@@ -303,6 +307,7 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
 
 
 class SpectralEmbedding(BaseEstimator):
+
```

Comment:
Kindly remove this line... I think you have used autopep8 right? We do not usually use blank lines before docstrings... :)